controllers/hiringRequestController.js

```diff
@@ -38,26 +38,43 @@ const createHiringRequest = async (req, res) => {
     // Create a notification for the teacher
     const notification = new Notification({
       userId: teacherId,
-      message: 'You have a new hiring request',
-      eventType: 'new_hiring_request',
+      message: "You have a new hiring request",
+      eventType: "new_hiring_request",
       eventDetails: { requestId: newRequest._id },
     });
 
     // Save the notification to the database
     await notification.save();
 
     // Send an email notification to the teacher (optional)
-    const { email } = await Tutor.findOne({ _id: teacherId });
```

Comment:
Change this one to back  @Sohaib56 
Tutor.findOne({ _id: teacherId });

Reply:
const { email, name: tutorName } = Tutor.findOne({ _id: teacherId });
just paste this into cod